src/index.js

```diff
@@ -294,6 +294,7 @@ the MQ_MAKE_EXCHANGES environment variable and restart.
                 routingKey: message.fields.routingKey,
                 correlationId: message.properties.correlationId,
                 timestamp: message.properties.timestamp,
+                expiresAfter: exchangeGroup.retryQueue.perMessageTtl || undefined,
```

Comment:
This means the "per message ttl" is not in fact per message, right? It's just using a different way to execute the configured ttl?

Reply:
Yes it's a different way to use the configured ttl, however, instead of setting it on a queue (that we cannot change later), this will set on individual message (that we can change).